fem/fespace.hpp

```diff
@@ -756,6 +756,9 @@ class QuadratureSpace
    /// Return the total number of quadrature points.
    int GetSize() const { return size; }
 
+   /// Return the order used by the global rules from #IntRules.
```

Comment:
I find the description somewhat unclear. Maybe use something like: "Return the order of the quadrature rules used by all elements."